gdcdictionary/schemas/diagnosis.yaml

```diff
@@ -18063,6 +18125,9 @@ properties:
         $ref:
           - "_terms_enum.yaml#/not_reported/common"
 
+  sites_of_involvement_count:
+    type: integer
+
```

Comment:
Add "min: 1" here?  @tlicht3

Reply:
done 👍

Reply:
Add minimum, can remove if decided not needed

Reply:
@wwysoc2 and @bullenc I think this is fine... I don't think folks should submit anything to this property if there is not involvement/no cancer.